packages/documentation-site/patternfly-docs/content/get-started/design-with-figma.md

```diff
@@ -0,0 +1,89 @@
+---
+id: Design
+title: Design with PatternFly
+section: get-started
+source: Design-with-Figma
+
+---
+import './get-started.css';
+import {Button, Divider} from '@patternfly/react-core';
+import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-icon';
+
+
+## Penta design kit
+
+To start designing with PatternFly, you will need to install our current design kit, called "Penta". This kit gives you access to PatternFly's visual design system — including design tokens — so that you can create consistent and usable product experiences. 
+
+<div style='text-align:center;'>
```

Comment:
I think this could be aligned left with the rest of the content. @lboehling you cool with that?

Reply:
cool w it!